src/playwright-runner.ts

```diff
@@ -401,14 +406,15 @@ async function runPlaywright(nodeBin, runCfg) {
     excludeParams.push('browser');
   }
 
+  // eslint-disable-next-line prefer-const
   for (let [key, value] of Object.entries(args)) {
     key = utils.toHyphenated(key);
     if (excludeParams.includes(key.toLowerCase()) || value === false) {
       continue;
     }
     procArgs.push(`--${key}`);
     if (value !== true) {
-      procArgs.push(value);
+      procArgs.push(new String(value).toString());
```

Comment:
I _think_ this can be shortened
```suggestion
      procArgs.push(String(value));
```